spec/support/factories/properties.rb

```diff
@@ -2,10 +2,8 @@
 
 FactoryBot.define do
   factory :property do
-    name { 'Lakeview Manor' }
+    game
 
-    trait :with_matching_canonical do
```

Comment:
This isn't being used anywhere because the `Property` tests actually sync canonicals, preventing more from being created, and it doesn't seem to be used elsewhere either.